### PR TITLE
PGSRETUR6247- Added method for calculation 

### PR DESCRIPTION
**Pull request by Indrani55**
Hey I have resolved the issue of calculation.

Regards,
Motu Shukla
01101001 00100000 01101100 01101111 01110110 01100101 00100000 01111001 01101111 01110101

### CODE REVIEW

**Comment by Indrani55** [comment]
Hey I have resolved the issue of calculation.

Regards,
Motu Shukla
01101001 00100000 01101100 01101111 01110110 01100101 00100000 01111001 01101111 01110101

**Comment by Indrani55** [comment]
Hey I have resolved the issue of calculation.

Regards,
Motu Shukla
01101001 00100000 01101100 01101111 01110110 01100101 00100000 01111001 01101111 01110101

File: A [hunk 1]
```diff
@@ -2,4 +2,8 @@ Class A{
  public static void main(String[] args){
    System.out.println("execute A");
  }
+public string abs(){
```
Comment (devavrata) [inline A]: @Indrani55 Can you please add comment on this method.

File: A [hunk 2]
```diff
@@ -2,4 +2,8 @@ Class A{
...
+String s=“dev”;
```
Comment (devavrata) [inline A]: Can you please rename the string variable and use camelcase while naming it.

**Review by devavrata — CHANGES_REQUESTED** [review]
Please resolve reviews.

**Comment by Indrani55** [comment]
Can you please recheck it

**Comment by Indrani55** [comment]
@devavrata recheck

**Comment by Indrani55** [comment]
@devavrata  check now

**Review by devavrata — APPROVED** [review]

**Pull request merged**